src/auth/authenticator/oicd/testutils/jwtGenerator.go

```diff
@@ -0,0 +1,55 @@
+package testutils
+
+import (
+	"crypto/rsa"
+	"encoding/json"
+	"strings"
+	"time"
+
+	"github.com/consensys/quorum-key-manager/pkg/tls/certificate"
+	"github.com/consensys/quorum-key-manager/src/auth/authenticator/oicd"
+	"github.com/golang-jwt/jwt"
+)
+
+type JWTGenerator struct {
+	privateKey *rsa.PrivateKey
+	claims     *oicd.ClaimsConfig
+}
+
+func NewJWTGenerator(keyPair *certificate.KeyPair, claims *oicd.ClaimsConfig) (*JWTGenerator, error) {
+	cert, err := certificate.X509(keyPair)
+	if err != nil {
+		return nil, err
+	}
+
+	return &JWTGenerator{
+		privateKey: cert.PrivateKey.(*rsa.PrivateKey),
+		claims:     claims,
+	}, nil
+}
+
+func (j *JWTGenerator) GenerateAccessToken(username string, groups []string, ttl time.Duration) (tokenValue string, err error) {
```

Comment:
Do we need this in the codebase?

Reply:
it is part of our CMD utils to generate tokens to work with IMO it is useful to provide such a thing and it is not part of our src but pkg what I believe it is good since it does to mix with the app codebase